samples/bluetooth/broadcast_audio_sink/README.rst

```diff
@@ -20,5 +20,7 @@ Building and Running
 ********************
 This sample can be found under
 :zephyr_file:`samples/bluetooth/broadcast_audio_sink` in the Zephyr tree.
+Use `-DOVERLAY_CONFIG=overlay-bt_ll_sw_split.conf` to enable required ISO
+feature support in Zephyr Bluetooth Controller on supported boards.
```

Comment:
How does a user tell if a board is supported?

Reply:
I don't think so. Since there are >500 platforms in zephyr it is done rather the other way around.  We define on a sample level (in sample.yaml) what are requirements to be able to execute the sample on a given board (e.g. `filter`, `min_ram`, `depends_on` or `platform_allow`).  In NCS we have a smaller set of nrf platforms and maintainers explicitly defines which platforms are supported in a sample.yaml and when documentation is build those entries are parsed and a table is generated (e.g. https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/crypto/aes_cbc/README.html)
In case of the sample from the PR there is a strictly defined set of platforms due to `platform_allow` usage (twister will block anything outside of it from building by default). Does this mean those are the only supported platforms?